apps/core/scenes/search/PlacePicker/PlacePickerContent.js

```diff
@@ -83,10 +83,11 @@ class PlacePickerContent extends React.Component<Props, State> {
         <View>
           <Text weight="bold">{this.getLabel()}:</Text>
           <TextInput
-            placeholder="write a place..."
+            placeholder="Write a place..."
             onChangeText={this.handleChangeText}
             value={this.state.text}
             prefix={<Icon name="search" />}
+            autoCorrect={false}
```

Comment:
It doesn't seem to work. Sadly we are using `universal-components` version of `TextInput`.
Where this props is not passed down. So we will need to update it first.

Reply:
I will do that, I currently working on autoFocus for TextInput so this should be easy.

Reply:
cf https://github.com/kiwicom/universal-components/pull/245

Reply:
done